src/transformers/models/deprecated/mctct/configuration_mctct.py

```diff
@@ -80,17 +80,17 @@ class MCTCTConfig(PretrainedConfig):
             The probability of randomly dropping the `Conv1dSubsampler` layer during training.
         num_conv_layers (`int`, *optional*, defaults to 1):
             Number of convolution layers before applying transformer encoder layers.
-        conv_kernel (`List[int]`, *optional*, defaults to `[7]`):
+        conv_kernel (`List[int]`, *optional*, defaults to `(7,)`):
```

Comment:
Haha it accepts `List[int]` but refuses to keep the list as the default

Reply:
IIUC this doesn't change the expected types, only the default, so maybe worth updating the type hint here

Reply:
Will GitHub refuses me to add a hint, but wanted to offer a change from `List` to `Sequence`